lib/subproviders/geth_api_double.js

```diff
@@ -545,7 +545,18 @@ GethApiDouble.prototype.evm_setTime = function(date, callback) {
   callback(null, this.state.blockchain.setTime(date));
 };
 
-GethApiDouble.prototype.evm_mine = function(timestamp, callback) {
+
+GethApiDouble.prototype.evm_mineBlockNumber = function (number, callback) {
+  var self = this;
+  this.state.blockchain.setBlockNumber(number)
```

Comment:
Should this be here?

Reply:
yes, 
1. set the block number to what we want, 
2. mine a block, 
3. unset the adjustment